kse/src/org/kse/gui/actions/FindAction.java

```diff
@@ -53,7 +53,7 @@ protected void doAction() {
 				Set<String> aliases = findEntryAlias(name); 
 				if (aliases.isEmpty()) {
 					JOptionPane.showMessageDialog(frame,
-							dialog.getEntryName() + " " + res.getString("FindAction.NotFound.message"),
+							MessageFormat.format(res.getString("FindAction.NotFound.message"), name),
```

Comment:
The import for MessageFormat is missing

Reply:
_Sorry, I am not a java dev._

Is `import java.text.MessageFormat;` it enough?